### ISSUE TITLE
[request] a simple way to expose the rownumbers that fail for a rule

### ISSUE THREAD

**Issue by RMHogervorst** [data-cleaning/validate#70]
I can't find a simple way to view only the rows I need. 

values(cf) # leads to a nice printed overview but I want to return the rownumbers that fail for instance.

Some more documentation about this would be nice too

**Comment by RMHogervorst** [comment]
I now understand that values(cf) 
leads to a list of 2. 
part 1: rules that confront the entire dataset
part 2: rules per row. 

So subsetting the list like this
```
result <- values(cf)
result[[2]]
```

leads to the matrix of rules per row. 
So (with dplyr) my own answer would be:

```
result[[2]] %>% as.data.frame() %>%  filter(V7 ==FALSE)
```

Is there a better way?

**Comment by markvanderloo** [comment]
I tried to reproduce this using the following code
```r
library(dplyr)
library(magrittr)
library(validate)
cf <- check_that(iris
  , mean(Sepal.Length)>0
  , Sepal.Length > 5.3 )

results <- values(cf)[[2]]

results %>% as.data.frame() %>% filter(V2==FALSE)
```

however I do not get the error you get. Could you set up a small example reproducing this behaviour?

**Comment by markvanderloo** [comment]
by the way, better facilities for analyzing results is definitely on our list. The fact that results vary in dimensional structure is the main difficulty.

**Comment by RMHogervorst** [comment]
indeed your example does not throw errors.
